wildfly/connectors/cassandra/feature-pack-cassandra/wildfly-user-feature-pack-build.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" ?>
+
+<build xmlns="urn:wildfly:feature-pack-build:3.1" producer="org.jboss.teiid:galleon-pack-cassandra">
+    <transitive>
+        <dependency group-id="org.wildfly" artifact-id="wildfly-ee-galleon-pack">
+            <name>org.wildfly:wildfly-ee-galleon-pack</name>
+            <packages inherit="false">
+                <exclude name="product.conf"/>
+            </packages>
+            <default-configs inherit="false"/>
+        </dependency>
+    </transitive>
+    <dependencies>
+<!--        <dependency group-id="org.wildfly" artifact-id="wildfly-galleon-pack">-->
```

Comment:
Could you remove this commented out config if it's not needed?

Reply:
Removed the comment.